src/screens/ReceiveFunds.js

```diff
@@ -45,7 +39,7 @@ export default class ReceiveFunds extends Component<*, *> {
 
   componentDidMount() {
     const { params } = this.props.navigation.state;
-    let amount = 0;
+    const amount = 0;
```

Comment:
I think there is a bug here, the internal `let amount` shouldn't be a variable but the same. anyway this part is likely temporary

Reply:
not sure I get what you mean. do you think it should be a `let` ?

Reply:
that's ok. there is just a broken logic in the code. that whole block is literally useless https://github.com/LedgerHQ/ledger-wallet-mobile/blob/master/src/screens/ReceiveFunds.js#L49-L54 because accidentally have an inner let amount, where it should just be an assignment, not a declaration